client/src/pages/Videopage/Videopage.module.css

```diff
@@ -0,0 +1,39 @@
+.videosection {
```

Comment:
Check naming conventions regarding css class
_https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/_